nb2kg/_version.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-version_info = (0, 0, 1, 'dev')
+version_info = (0, 0, 2)
```

Comment:
Maybe this should move to a 0.1.0 and leave the patch part of the version number for bug fixes to that release.

Reply:
Great idea - done.